third_party/houdini/lib/gusd/CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ pxr_shared_library(${PXR_PACKAGE}
         usdRi
         usdShade
         usdUtils
+        ${HOUDINI_OPZ_LIBRARY}
+        ${HOUDINI_UI_LIBRARY}
```

Comment:
I wasn't able to find any documentation about what the various Houdini .so's provide, but our FX team pointed out that the Makefile included in the Houdini dev kit under toolkit/makefiles/Makefile.linux links against:

`-lHoudiniUI -lHoudiniOPZ -lHoudiniOP3 -lHoudiniOP2 -lHoudiniOP1 -lHoudiniSIM -lHoudiniGEO -lHoudiniPRM -lHoudiniUT`

Should we do the same, even if OPZ and UI happen to be sufficient for now?

Reply:
Sounds good. That would make it development more future proof, especially if we start relying on more API functions. Loading libraries and such outside houdini is more of a rare thing, so it's unlikely it would be tested by houdini devs.